src/test/scala/org/ergoplatform/explorer/db/dao/InputsDaoSpec.scala

```diff
@@ -46,7 +46,8 @@ class InputsDaoSpec extends FlatSpec with Matchers with BeforeAndAfterAll with P
     val withValues = inputs.map { i =>
       val v = outputs.find(_.boxId == i.boxId).map(_.value).getOrElse(0L)
```

Comment:
Does it worth extracting `outputs.find(_.boxId == i.boxId)` to a variable and then reusing?

Reply:
@mike-aksarin I think we can afford such things in tests =)

Reply:
Ah, i missed this